### ISSUE TITLE
Add a noZoom option to disable zoom entirely.

### ISSUE THREAD

**Issue by szanni** [henrygd/bigger-picture#29]
It would be beautiful to have a `noZoom` option that disabled zooming entirely.

Currently one can seemingly inhibit zooming via a combination of `noPinch` & `maxZoom: 1` or `noPinch` & `onImageClick`.
```js
bp.open({
  noPinch: (container) => true,
  maxZoom: 1,
  onImageClick: (inlineContainer, inlineActiveItem) => true,
})
```
Both `onImageClick` and `maxZoom: 1` do the same in my use case at least.

The problem that remains however is that the mouse cursor still displays a `cursor: zoom-in` on desktop when hovering over the opened gallery.
This is very confusing to the user and suggests something is broken, when it is just intended behavior.
Hence I believe adding a `noZoom` option would be a great addition to the library. (Thanks for your great work btw!)

**Comment by henrygd** [comment]
Seems like `noZoom: true` should be equivalent to `maxZoom: 1` so if that's not the current behavior I'll take a look and see what's going on.

The cursor thing is a bug that I've been meaning to fix for a while. It shows a zoom icon even if the image is small and unable to be zoomed.

ETA for this is some time next week probably, but I'm moving next month so I'm kind of busy.

For the time being you should be able to do something like this:

```js
onOpen(container) {
    container.classList.add('default-cursor')
}
```

```css
.default-cursor .bp-img {
    cursor: default;
}
```

**Comment by szanni** [comment]
Thanks for the promt response.

> Seems like noZoom: true should be equivalent to maxZoom: 1 so if that's not the current behavior I'll take a look and see what's going on.

`maxZoom: 1` does indeed inhibit zooming on desktop, I haven't tried if `noPinch` is even needed for mobile. I believe a remark in the docs under `maxZoom` would be helpful, something along the lines of `Setting maxZoom to 1 will disable zooming entirely.`.

>The cursor thing is a bug that I've been meaning to fix for a while. It shows a zoom icon even if the image is small and unable to be zoomed.

Ok, so this is a known issue. I'm currently just overriding the general CSS, as I have no use for zooming at all atm. A proper fix would be great though. Would do it myself, but it wasn't really obvious to me where to start.

> ETA for this is some time next week probably, but I'm moving next month so I'm kind of busy.

Have a good move and thanks again for your amazing work. Very impressed by the snappiness of the lib.

**Comment by henrygd** [comment]
Thanks!

Cursor issue is fixed in 1.1.9 and I added clarification to the readme that `maxZoom: 1` will completely disable zoom.

Let me know if you have questions about anything else.
